geomstats/geometry/general_linear.py

```diff
@@ -108,13 +108,13 @@ def log(cls, point, base_point=None):
 
         Parameters
         ----------
-        point :         array-like, shape=[..., n, n]
-        base_point :    array-like, shape=[..., n, n]
+        point : array-like, shape=[..., n, n]
```

Comment:
do we use ... now as well as n_samples?

Reply:
Good question. I reverted to shape=[n_samples, n, n] or shape=[n, n] since this is the convention of most of the codebase, this is more explicit anyway, and the goal of this PR is not to change this.